### PR TITLE
updated readme and project page

### PR DESCRIPTION
**Pull request by johnw335**

### CODE REVIEW

File: projects/smart-classroom.md [hunk 1]
```diff
@@ -9,10 +9,20 @@ While taking classes in Ithaca during the summer, participation was a significan
 
 This made me realize classrooms haven't changed except for maybe the projector for a very long time, and utilize no technology.
 
-### Challenge
 
-To make a 'smart classroom' leveraging technology such as beacons and cameras (facial recognition, eye direction, body movements, facial expressions, etc) to change the way participation is graded, teachers get feedback, etc etc
+### Story:
+(For September 23rd)
```
Comment (dblock) [inline projects/smart-classroom.md]: The story should be in a project management tool like Trello or Waffle.io. Could you please take it out of here?

**Comment by dblock** [comment]
Bump @johnw335

**Comment by johnw335** [comment]
Changes made. Thanks

**Comment by dblock** [comment]
Notice the "This branch has conflicts that must be resolved." here. You need to rebase this branch. Ask @karnseth or @dagoch to help you if you can't figure it out.

**Comment by dblock** [comment]
Closing, no activity.

**Pull request closed without merging**